#! /bin/sh /usr/share/dpatch/dpatch-run
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Make sure the links in the only documentation still work.

@DPATCH@
diff -urNad phpmyadmin-3.1.0~/Documentation.html phpmyadmin-3.1.0/Documentation.html
--- phpmyadmin-3.1.0~/Documentation.html	2008-11-28 14:21:00.000000000 +0100
+++ phpmyadmin-3.1.0/Documentation.html	2008-11-30 12:24:59.000000000 +0100
@@ -43,7 +43,6 @@
             Official phpMyAdmin wiki</a></li>
     <li>Local documents:
         <ul><li>Version history: <a href="changelog.php">ChangeLog</a></li>
-            <li>General notes: <a href="readme.php">README</a></li>
             <li>License: <a href="license.php">LICENSE</a></li>
         </ul>
     </li>
diff -urNad phpmyadmin-3.1.0~/changelog.php phpmyadmin-3.1.0/changelog.php
--- phpmyadmin-3.1.0~/changelog.php	2008-11-28 14:21:00.000000000 +0100
+++ phpmyadmin-3.1.0/changelog.php	2008-11-30 12:31:24.000000000 +0100
@@ -6,7 +6,21 @@
  * @version $Id: changelog.php 10815 2007-10-17 08:02:56Z cybot_tm $
  */
 
-$changelog = htmlspecialchars(file_get_contents('ChangeLog'));
+/**
+ * Path to config file, can be gzip compressed.
+ */
+define('CHANGELOG_FILE', '/usr/share/doc/phpmyadmin/changelog.gz');
+
+if (substr(CHANGELOG_FILE, -3) == '.gz') {
+    ob_start();
+    readgzfile(CHANGELOG_FILE);
+    $changelog = ob_get_contents();
+    ob_end_clean();
+} else {
+    $changelog = file_get_contents(CHANGELOG_FILE);
+}
+
+$changelog = htmlspecialchars($changelog);
 
 $replaces = array(
     '@(http://[./a-zA-Z0-9.-]*[/a-zA-Z0-9])@'
diff -urNad phpmyadmin-3.1.0~/license.php phpmyadmin-3.1.0/license.php
--- phpmyadmin-3.1.0~/license.php	2008-11-28 14:21:00.000000000 +0100
+++ phpmyadmin-3.1.0/license.php	2008-11-30 12:24:59.000000000 +0100
@@ -13,5 +13,5 @@
  *
  */
 header('Content-type: text/plain; charset=iso-8859-1');
-readfile('LICENSE');
+readfile('/usr/share/doc/phpmyadmin/copyright');
 ?>
